### ISSUE TITLE
How do you train deeplab v2 with full supervised method using your pseudo label?

### ISSUE THREAD

**Issue by TyroneLi** [Juliachang/SC-CAM#5]
DO you change any parameters or model architecture using this [repro](https://github.com/kazuto1011/deeplab-pytorch#pascal-voc-2012)?

**Comment by zerodohero** [comment]
I'd like to know

**Comment by wasidennis** [comment]
@TyroneLi @zerodohero The model architecture is the same. We did not heavily tune the model but just lower the learning rate a bit to account for noisy pseudo labels.

**Comment by kkechen** [comment]
Sorry to bother you, @wasidennis could you share the modified parameter e.g. learning rate of the segmentation network. That would be helpful for us to re-implement your work, thanks
